src/controllers/dynakube/token/tokens.go

```diff
@@ -0,0 +1,112 @@
+package token
+
+import (
+	"fmt"
+	"strings"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/dtclient"
+	"github.com/pkg/errors"
+)
+
+type Tokens map[string]Token
+
+func (tokens Tokens) ApiToken() Token {
+	return tokens.getToken(dtclient.DynatraceApiToken)
+}
+
+func (tokens Tokens) PaasToken() Token {
+	return tokens.getToken(dtclient.DynatracePaasToken)
+}
+
+func (tokens Tokens) DataIngestToken() Token {
+	return tokens.getToken(dtclient.DynatraceDataIngestToken)
+}
+
+func (tokens Tokens) getToken(tokenName string) Token {
+	token, hasToken := tokens[tokenName]
+
+	if !hasToken {
+		return Token{}
```

Comment:
We can avoid multiple returns here
```suggestion
		token = Token{}
```

Reply:
done